bundles/org.openhab.core.ephemeris/src/main/java/org/openhab/core/ephemeris/internal/EphemerisManagerImpl.java

```diff
@@ -112,12 +113,21 @@ public EphemerisManagerImpl(final @Reference LocaleProvider localeProvider) {
             final Properties properties = new Properties();
             properties.load(stream);
             properties.forEach(this::parseProperty);
+
+            sortByLabel(countries);
```

Comment:
I had difficulties figuring out that the line above actually fills the Lists of the Maps with values by calling `parseProperty` and that those Lists are `ArrayList` instances that are sortable. Imho it is a pretty unexpected side-effect of a method with the name `parseProperty`...

But that's not the fault of your PR, so all fine for merging this!

Reply:
I would also not expect these side effects as the current name suggests it's some kind of trivial operation. ;-)